### ISSUE TITLE
Client doesn't seem to fall

### ISSUE THREAD

**Issue by DigitalSniperz** [ORelio/Minecraft-Console-Client#170]
When you break a block under the client, it doesn't seem to fall. I've tried /move down and /move up. Heres output of those, 

```
>/move down
MCC: Moving down. (Doesn't move down)
>/move up
MCC: Cannot move in that direction.
>/move east
MCC: Moving east. (Still doesn't move)
```

Is there anyway we can make the client fall on its own or using a command?

**Comment by ORelio** [comment]
Seems like the block break event wasn't caught properly or an anti-cheat plugin is interfering.
Did you try on a server you own an can guarantee there wasn't such a plugin?

**Comment by DigitalSniperz** [comment]
Theres no block break event, the bot spawns in the server flying in the air but I want the bot to fall down because theres no block under him.

Tested with no anticheat plugin, no plugins, still the issue occurs.

**Comment by ORelio** [comment]
Ah, so even without break event that doesn't work, okay. Which Minecraft version?

**Comment by DigitalSniperz** [comment]
Latest version of the client (from dev builds) and minecraft version 1.9.2.

**Comment by ORelio** [comment]
Will take a look at this. Thanks for your report.

**Comment by DigitalSniperz** [comment]
Maybe you could check on server join to see if the block under the player is air, if so make them fall.

**Comment by ORelio** [comment]
Falling is already automatic when terrain and movements is enabled. Well, when it works.

**Comment by DigitalSniperz** [comment]
Weird, it appears sometimes the client falls when it joins the server.

**Comment by ORelio** [comment]
Strange, falling was tested OK on 1.9.4 while testing #173 - Cannot reproduce :sweat:

**Comment by ORelio** [comment]
Issues with anti-cheat plugins should have been solved now, see #195 for more details :)
